### PR TITLE
more parse jets

### PR DESCRIPTION
**Pull request by matthew-levan**

### CODE REVIEW

File: rust/ares/Cargo.toml [hunk 1]
```diff
@@ -12,9 +12,9 @@ edition = "2018"
 # Please keep these alphabetized
 [dependencies]
 ares_macros = { path = "../ares_macros" }
-assert_no_alloc = "1.1.2"
+# assert_no_alloc = "1.1.2"
 # use this when debugging requires allocation (e.g. eprintln)
-# assert_no_alloc = {version="1.1.2", features=["warn_debug"]}
+assert_no_alloc = {version="1.1.2", features=["warn_debug"]}
```
Comment (eamsden) [inline rust/ares/Cargo.toml]: Please put this back before merging @matthew-levan

File: rust/ares/src/jets/parse.rs [hunk 2]
```diff
@@ -255,12 +255,9 @@ pub fn jet_stir(context: &mut Context, subject: Noun) -> Result {
...
                 let mut p_vex = vex.head();
                 let mut q_vex = vex.tail();
-                eprintln!("stir: got vex\r\n");
-                let i = 0;
                 while !q_vex.raw_equals(D(0)) {
-                    eprintln!("stir: starting vex loop {}\r\n", i);
-                    let puq_vex = q_vex.as_cell()?.head();
-                    let quq_vex = q_vex.as_cell()?.tail();
+                    let puq_vex = slot(q_vex, 6)?;
+                    let quq_vex = slot(q_vex, 7)?;
```
Comment (eamsden) [inline rust/ares/src/jets/parse.rs:260]: Flagging that this was the bugfix for the actual jet logic

**Comment by eamsden** [comment]
I'm moving this back to draft because it's not clear it actually improves anything.

**Comment by ashelkovnykov** [comment]
Appears all clear from my end, after latest changes. Boot times:
Before = `12m 00s`
After = `11m 06s`

**Comment by ashelkovnykov** [comment]
@eamsden Ping - this PR is good to merge, pending your review

**Review by eamsden — APPROVED** [review]

**Pull request merged**